pkg/kv/bulk/sst_batcher_test.go

```diff
@@ -351,3 +353,75 @@ func runTestImport(t *testing.T, batchSizeValue int64) {
 		})
 	}
 }
+
+var DummyJobID int64 = 9999
+
```

Comment:
the test below covers a subset of the test in import_mvcc_test.go. I include in case we want to divide this PR.